src/routes-data/sidebar-routes.jsx

```diff
@@ -178,29 +116,45 @@ const ImplementDropdown = route => {
 
 function LinkComponent({ route, path }) {
     const { pathname } = useLocation();
+    const split_pathname = pathname.split('/');
+    const split_path = path.split('/');
+
+    const last_pathname = split_pathname[split_pathname.length - 1] !== '' 
+    ? split_pathname[split_pathname.length - 1]
+    : split_pathname[split_pathname.length - 2];
+
+    const last_path = split_path[split_path.length - 1] !== '' 
+    ? split_path[split_path.length - 1]
+    : split_path[split_path.length - 2];
+
+    const path_is_pathname = last_pathname === last_path;
```

Comment:
```suggestion
    const getLastPathString = (split_path_or_pathname) => {
        return split_path_or_pathname[split_path_or_pathname.length - 1] !== '' 
            ? split_path_or_pathname[split_path_or_pathname.length - 1]
            : split_path_or_pathname[split_path_or_pathname.length - 2];

    const path_is_pathname = getLastPathString(pathname.split('/')) === getLastPathString(path.split('/'));
```
🙂 just a way to make it shorter

Reply:
Cool, thanks! Implemented